### DISCUSSION TITLE
hide python file path in vscode terminal

### DISCUSSION THREAD

**Discussion by RicardoMSRocha** [microsoft/vscode-discussions#528] [Extension Development QnA]
Does anyone know how do I hide the python file path in the vscode terminal as shown in the image:

![Captura de tela 2023-03-16 215412](https://user-images.githubusercontent.com/110740786/225784536-9589733b-0e7f-4520-9011-aa5231bb4ed3.png)

**Comment by isidorn** [comment]
Please ask on StackOverflow
